cmd/allocate/allocate.go

```diff
@@ -80,7 +80,7 @@ func (c *allocateCommand) Init(args []string) error {
 	var err error
 	c.Budget, c.Limit, err = parseBudgetWithLimit(budgetWithLimit)
 	if err != nil {
-		return err
+		return errors.Annotate(err, `expected args in the form "service budget:limit"`)
```

Comment:
hmm.. yes, i like it.. but maybe we should display the help message if/when something is wrong with arguments..